v3/src/components/graph/adornments/univariate-measures/standard-deviation/standard-deviation-adornment-model.ts

```diff
@@ -0,0 +1,31 @@
+import { Instance, types } from "mobx-state-tree"
+import { mean, std } from "mathjs"
+import {
+  UnivariateMeasureAdornmentModel, IUnivariateMeasureAdornmentModel
+} from "../univariate-measure-adornment-model"
+import { kStandardDeviationKey, kStandardDeviationType } from "./standard-deviation-adornment-types"
+import { IDataConfigurationModel } from "../../../../data-display/models/data-configuration-model"
+
+export const StandardDeviationAdornmentModel = UnivariateMeasureAdornmentModel
+  .named("StandardDeviationAdornmentModel")
+  .props({
+    type: "Standard Deviation",
+    labelTitle: types.optional(types.literal(kStandardDeviationKey), kStandardDeviationKey)
+  })
+  .views(self => ({
+    computeMeasureRange(attrId: string, cellKey: Record<string, string>, dataConfig: IDataConfigurationModel) {
+      const caseValues = self.getCaseValues(attrId, cellKey, dataConfig)
+      return std(caseValues)
```

Comment:
Have you verified that the MathJS implementation handled edge cases like no cases or only one case the way we would like? That's always the potential downside of delegating to another library.

Reply:
@kswenson I believe so. It seems to match V2's behavior. With no cases, no univariate adornments will appear (except for Plotted Value) when their options are checked. With one case, the Mean, Median, SD, and MAD will appear but they are all a single line. SD and MAD do not show a range.